src/states/common/base.rs

```diff
@@ -99,6 +115,21 @@ pub trait Base: Display {
         Err(InterfaceError::InvalidState)
     }
 
+    fn handle_send_msg_to_client(
+        &mut self,
+        _peer_addr: SocketAddr,
+        _msg: NetworkBytes,
+        _token: Token,
+    ) -> Result<(), InterfaceError> {
+        warn!("{} - Cannot send message to client- invalid state.", self);
+        Err(InterfaceError::InvalidState)
```

Comment:
Why not just send the message here? You don't need to be elder to do it.

Reply:
You are sending to a client. It doens't make sense unless you are an elder